### PR TITLE
chore: streamline db initialization make commands

### PR DESCRIPTION
**Pull request by BalduinLandolt**
### Task Description/Number

Issue Number: DEV-

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [ ] fix: represents bug fixes
- [ ] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [x] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [ ] Yes (don't forget to update the JS-LIB team about the change)
- [ ] No

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -255,57 +254,55 @@ init-db-test-unit-minimal: env-file stack-down-delete-volumes stack-db-only ## i
...
+init-db-from-prod: ## init local database with data from prod server. Use as `make init-db-from-prod PW=database-password`
 	@echo $@
-	@curl -X POST -H "Content-Type: application/sparql-update" -d "DROP ALL" -u "admin:test" "http://localhost:3030/knora-test"
-	@curl -X POST -H "Content-Type: application/trig" -T "${CURRENT_DIR}/db_test_dump.trig" -u "admin:test" "http://localhost:3030/knora-test"
+	${MAKE} init-db-from-env ENV=db.dasch.swiss
 
-.PHONY: init-db-test-from-staging
-init-db-test-from-staging: db_staging_dump.trig init-db-test-empty ## init local database with data from staging
+.PHONY: init-db-from-prod-dump
+init-db-from-prod-dump: ## init local database with data from local dump file of prod server. Use as `make init-db-from-prod-dump
```
Comment (irinaschubert) [inline Makefile:283]:
```suggestion
init-db-from-prod-dump: ## init local database with data from local dump file of prod server. Use as `make init-db-from-prod-dump`
```

File: Makefile [hunk 2]
```diff
@@ -255,57 +254,55 @@ init-db-test-unit-minimal: env-file stack-down-delete-volumes stack-db-only ## i
...
+.PHONY: init-db-from-test
+init-db-from-test: ## init local database with data from test server. Use as `make init-db-from-test PW=database-password`
 	@echo $@
-	@curl -X GET -H "Accept: application/trig" -u "admin:${DB_TEST_PASSWORD}" "https://db.test.dasch.swiss/dsp-repo" > "db_test_dump.trig"
+	${MAKE} init-db-from-env ENV=db.test.dasch.swiss
 
-## Dump staging data
-db_staging_dump.trig:
+.PHONY: init-db-from-test-dump
+init-db-from-test-dump: ## init local database with data from local dump file of test server. Use as `make init-db-from-test-dump
```
Comment (irinaschubert) [inline Makefile:263]:
```suggestion
init-db-from-test-dump: ## init local database with data from local dump file of test server. Use as `make init-db-from-test-dump`
```

File: Makefile [hunk 3]
```diff
@@ -255,57 +254,55 @@ init-db-test-unit-minimal: env-file stack-down-delete-volumes stack-db-only ## i
...
+init-db-from-dev: ## init local database with data from dev. Use as `make init-db-from-dev PW=database-password`
 	@echo $@
-	@curl -X POST -H "Content-Type: application/sparql-update" -d "DROP ALL" -u "admin:test" "http://localhost:3030/knora-test"
-	@curl -X POST -H "Content-Type: application/trig" -T "${CURRENT_DIR}/db_prod_dump.trig" -u "admin:test" "http://localhost:3030/knora-test"
+	${MAKE} init-db-from-env ENV=db.dev.dasch.swiss
 
-.PHONY: init-db-test-from-dev
-init-db-test-from-dev: ## init local database with data from dev. Use as `make init-db-test-from-dev PW=database-password`
+.PHONY: init-db-from-dev-dump
+init-db-from-dev-dump: ## init local database with data from local dump file of dev server. Use as `make init-db-from-dev-dump
```
Comment (irinaschubert) [inline Makefile:293]:
```suggestion
init-db-from-dev-dump: ## init local database with data from local dump file of dev server. Use as `make init-db-from-dev-dump`
```

**Review by irinaschubert — APPROVED** [review]
just some typos :)

**Comment by BalduinLandolt** [comment]
> just some typos :)

the beauty of copy-paste: you make one mistake and it shows up X times... thanks for spotting

**Pull request merged**